src/components/FeaturedProducts.js

```diff
@@ -0,0 +1,59 @@
+
+import React from 'react'
+import featuredProducts from '../data/featured-products.json'
+import './Home.css'
+
+function FeaturedProducts(){
+    const products = featuredProducts.results;
+    const [productItems] = React.useState(products);
```

Comment:
I am going to suggest you to deconstruct on the first line the useState to avoid using 'React.' and directly using the hook

On the other hand I put a common way to call the useState

```suggestion
import React, {useState} from 'react'
import featuredProducts from '../data/featured-products.json'
import './Home.css'

function FeaturedProducts(){
    const [productItems, setProductItems] = useState(''); //This is the a common expression used in React for useState, with the variable and the function to set the data
    setProductItems(featuredProducts.results);
```

Reply:
Line 8 in suggestion cause render error, not sure if because setProductItems results in setting again the variable when it renders.

Reply:
Ok, then nevermind, only apply the deconstruction of the useState as a good practice

```suggestion
import React, { useState } from 'react'
import featuredProducts from '../data/featured-products.json'
import './Home.css'

function FeaturedProducts(){
    const products = featuredProducts.results;
    const [productItems] = useState(products);
```
```suggestion
import React from 'react'
import featuredProducts from '../data/featured-products.json'
import './Home.css'

function FeaturedProducts(){
    const products = featuredProducts.results;
    const [productItems] = React.useState(products);
```